zanata-common-api/src/main/java/org/zanata/rest/service/GlossaryResource.java

```diff
@@ -66,11 +69,34 @@
     @TypeHint(Glossary.class)
     public Response getEntries();
 
+
     /**
-     * Returns Glossary entries for a single locale.
+     * Return locales available for all glossary entries
+     *
+     * @return The following response status codes will be returned from this
+     *         operation:<br>
+     *         OK(200) - Response containing all Glossary entries in the system.
+     *         INTERNAL SERVER ERROR(500) - If there is an unexpected error in
+     *         the server while performing this operation.
+     */
+    @GET
+    @Path("/locales/list")
+    @Produces({
+        MediaTypes.APPLICATION_ZANATA_GLOSSARY_JSON,
+        MediaType.APPLICATION_JSON })
+    @TypeHint(LocaleDetails[].class)
```

Comment:
Same here is it meant to be GlossaryLocaleStats? And this method only returns JSON type but other methods supports both xml and json. Is it intentional? Since this is not a hidden server API I am not sure if this is good idea.